### DISCUSSION TITLE
Interesting Pi-based IoT controller

### DISCUSSION THREAD

**Discussion by billz** [RaspAP/raspap-webgui#944] [Show and tell]
Translated from Italian:

> "Rotilio is a Raspberry Pi based IoT controller for connected automation and home automation solutions. RS-485, Dual Ethernet, DIN-RAIL, Real Time Clock, MQTT, HTTPs, Websocket, all in one pre-configured, customizable, open source device."

![E5bSyufWUAA7IXt](https://user-images.githubusercontent.com/229399/124381619-a894f580-dcc3-11eb-94ac-a6233b07aa72.jpeg)

Pre-configured software includes Node-Red and RaspAP.

Link: https://www.techmakersiot.com/rotilio-iiot-controller

**Comment by whi4** [comment]
It looks like the techmakersiot.com link is broken.  Searching rotilio iot shows some older arduino-compatible boards on git  :  https://github.com/techmakers/rotilio.cc
